node/_next_tick.ts

```diff
@@ -1,14 +1,13 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+// deno-lint-ignore-file no-inner-declarations
```

Comment:
Perhaps we should still keep the declarations at the top level, but just make certain statements conditional?

Reply:
I'd prefer not - I want to save a reference to `Deno.core` in `core` constant and reuse that inside functions.